app/main/plugins/core/cerebro/version/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { search } from 'cerebro-tools'
+import icon from '../icon.png'
+
+// Settings plugin name
+const NAME = 'Cerebro Version'
+
+// Settings plugins in the end of list
+const order = 9
+
+// Phrases that used to find settings plugins
+const KEYWORDS = [
+  NAME,
+  'ver',
+  'version'
+]
+
+/**
+ * Plugin to show app settings in results list
+ *
+ * @param  {String} options.term
+ * @param  {Function} options.display
+ */
+const settingsPlugin = ({ term, display, config, actions }) => {
+  const found = search(KEYWORDS, term).length > 0
+
+  if (found) {
+    const results = [{
+      order,
+      icon,
+      title: NAME,
+      term: NAME,
+      getPreview: () => ( <div><strong>0.2.7</strong></div> )
```

Comment:
How hard would it be to also show a commit hash for this build? I wouldn't spend too much effort on this, but if it is easy, this info would help with bugs like #212, where it is not clear whether the code has a bug, or the code just didn't make it to the release.